x-pack/plugin/core/src/main/resources/synthetics-mappings.json

```diff
@@ -2,6 +2,25 @@
   "template": {
     "mappings": {
       "dynamic_templates": [
+        {
+          "match_ip": {
+            "match_mapping_type": "string",
+            "match": "ip",
+            "mapping": {
+              "type": "ip"
+            }
+          }
+        },
+        {
+          "match_message": {
+            "match_mapping_type": "string",
+            "match": "message",
+            "mapping": {
+              "type": "text",
+              "norms": false
+            }
+          }
+        },
```

Comment:
do we really need messages for synthetics?

Reply:
@andrewvc Do we need it?

Reply:
I asked @andrewvc  in a different channel, who told me that metrics datasets cannot have a message field.